### PR TITLE
add doc for sql server sink

### LINKED ISSUE #2259: Document: feat(connector): add SQL Server sink
This issue tracks the documentation update needed for the merged PR #17154.

Source PR URL: https://github.com/risingwavelabs/risingwave/pull/17154
Source PR Merged At: 2024-06-13T02:40:25Z

### PR DESCRIPTION
**Pull request by zwang28**
## Info

- **Description**

  close #2259

- **Notes**

  - [ Include any supplementary context or references here. ]

- **Related code PR**

  - [ Provide a link to the relevant code PR here, if applicable. ]

- **Related doc issue**
  
  Resolves [ Provide a link to the relevant doc issue here, if applicable. ]

## For reviewers

- **Preview**

  - [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **dev** version of the documentation. ]

- **Key points**

  - [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`emile-00`, `hengm3467`, & `WanYixian`).

### CODE REVIEW

File: docs/guides/sink-to-sqlserver.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+   connector='sqlserver',
+   connector_parameter = 'value', ...
+);
+```
+
+## Parameters
+
+| Parameter Names | Description |
+| --------------- | ---------------------------------------------------------------------- |
+| type | Required. Allowed values: `appendonly` and `upsert`. |
```
Comment (emile-00) [inline docs/guides/sink-to-sqlserver.md]:
```suggestion
| type | Required. Allowed values: `append-only` and `upsert`. |
```

**Comment by emile-00** [comment]
Please also add SQL Server as a sink to the sidebar (sidebars.js) under the Sinks category and to the Integrations page (rw-integration-summary.md) under the Other databases sections. Otherwise LGTM, thanks!

File: docs/guides/sink-to-sqlserver.md (hunk 1 above)
Comment (zwang28) [reply docs/guides/sink-to-sqlserver.md]: Fixed this one.

BTW search "Allowed values: `appendonly` and `upsert`" you'll find more.

File: docs/guides/sink-to-sqlserver.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+This guide describes how to sink data from RisingWave to Microsoft SQL Server.
+
+You can test out this process on your own device by using the `sqlserver_sink.slt` demo in the [`integration_test directory`](https://github.com/risingwavelabs/risingwave/tree/main/integration_tests) of the RisingWave repository.
+
+## Prerequisites
+
+Before sinking data from RisingWave to SQL Server, please ensure the following:
+
+- The SQL Server table you want to sink to is accessible from RisingWave.
+- Ensure you have an upstream materialized view or table in RisingWave that you can sink data from.
```
Comment (emile-00) [inline docs/guides/sink-to-sqlserver.md]:
```suggestion
- You have an upstream materialized view or table in RisingWave that you can sink data from.
```

**Review by emile-00 — APPROVED** [review]

**Pull request merged**